Cogs.Publishers/CsSchemaPublisher.cs

```diff
@@ -490,6 +553,48 @@ private bool Isboolintdoubleulong(string name)
             return false;
         }
 
+        private object SimpleToXml(string origDataTypeName, string name, string start, bool isList)
+        {
+            string indent = Environment.NewLine + "                ";
+            if (isList) { indent += "    "; }
+            if (origDataTypeName.ToLower().Equals("duration")){ return $"{start}.Add(new XElement(\"{name}\", {name}.Ticks));"; }
+            if (origDataTypeName.ToLower().Equals("datetime")) { return $"{start}.Add(new XElement(\"{name}\", {name}.ToString(\"yyyy-MM-dd\\\\THH:mm:ss.FFFFFFFK\")));"; }
+            if (origDataTypeName.ToLower().Equals("time")) { return $"{start}.Add(new XElement(\"{name}\", {name}.ToString(\"u\").Split(' ')[1]));"; }
+            if (origDataTypeName.ToLower().Equals("date")){ return $"{start}.Add(new XElement(\"{name}\", {name}.ToString(\"u\").Split(' ')[0]));"; }
+            if (origDataTypeName.ToLower().Equals("gyearmonth"))
+            {
+                return $"var ym = new XElement(\"yearmonth\");{indent}if ({name}.Item3 != null) {{ ym.Add(new XElement(" +
```

Comment:
year month is a built in schema type. It will be serialized as a single string such as:
 2001-10, 2001-10+02:00, 2001-10Z, 2001-10+00:00, -2001-10, or -20000-04